assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -197,6 +210,9 @@ export const controls = {
 	[ FETCH_MODULES ]: () => {
 		return API.get( 'core', 'modules', 'list', null, { useCache: false } );
 	},
+	[ REFETCH_AUTHENICATION ]: () => {
+		return API.get( 'core', 'user', 'authentication', null, { useCache: false, timestamp: Date.now() } );
```

Comment:
I'm not sure this is right. Looking at `API.get`, the `timestamp` argument here wouldn't currently do anything, it needs to be in the `data` parameter (which is currently `null`) in order to affect the request URL.

Reply:
@felixarntz Ah yes, I mistook the arguments there, thanks. I've moved the timestamp into the `data` arg, and added an `expect()` in the tests to make sure `timestamp` is included as a param for those GET requests to the auth endpoint. I also checked this locally, and confirmed the timestamp shows up in the GET request during the module deactivation. Let me know if there's anything else to button up here.